api/src/main/kotlin/com/openlattice/admin/AdminApi.kt

```diff
@@ -36,11 +38,14 @@ const val ID = "id"
 const val ID_PATH = "/{$ID}"
 const val NAME = "name"
 const val NAME_PATH = "/{$NAME}"
+const val WAREHOUSE_ID_PARAM = "WarehouseId"
```

Comment:
The convention is camel case.

```suggestion
const val WAREHOUSE_ID_PARAM = "warehouseId"
```